src/main/java/com/powsybl/network/conversion/server/dto/EquipmentInfos.java

```diff
@@ -110,6 +114,22 @@ public static Set<VoltageLevel> getVoltageLevels(@NonNull Identifiable<?> identi
         throw NetworkConversionException.createEquipmentTypeUnknown(identifiable.getClass().getSimpleName());
     }
 
+    public static Set<String> getEquipmentSubType(@NonNull Identifiable<?> identifiable) {
+        if (identifiable instanceof HvdcLine) {
+            HvdcLine hvdcLine = (HvdcLine) identifiable;
+            String hvdcType1 = hvdcLine.getConverterStation1().getHvdcType().toString();
+            String hvdcType2 = hvdcLine.getConverterStation2().getHvdcType().toString();
+
+            // Create a set and add both types
+            Set<String> hvdcTypes = new HashSet<>();
```

Comment:
```suggestion
            Set<String> hvdcTypes = new LinkedHashSet<>();
```
Better to keep order

Reply:
okey